ODE Solver

jongornet14 opened the pull request:
I created another LIF function that now uses the RK4 scheme.  I also
added the soft threshold and refractory period.

dlevenstein committed:
added example function for ode solver: WCadapt_run

Commit:
RK4 scheme

I created another LIF function that now uses the RK4 scheme.  I also
added the soft threshold and refractory period.

dlevenstein committed:
Merge branch 'master' into ODE_Solver

Brought updates to the AdLIFfunction to the ODE_Solver branch.

dlevenstein commented:
Awesome! 
Could you apply the changes I made to the Forwarrd Euler code to the RK4 code, and then go ahead and merge this branch into master.

Commit:
RK4 change

Made forward Euler and RK4 code the same in terms of solving the
differential equations

Commit:
AdLIF testing
